### ISSUE TITLE
3.8 Use HTML Elements Correctly (wording update suggestions)

### ISSUE THREAD

**Issue by andreadavanzo** [w3c/sustainableweb-wsg#20]
### 3.8 Use HTML Elements Correctly

HTML semantics are important. They don't just play a key role in making the Web look the way it does, they have a function in accessibility, SEO, and even in sustainability. Ensuring that you markup your content correctly and avoid cluttering your markup wastefully will reduce emissions.

**Success Criterion - Semantic Code**
Content must be marked up semantically using the right HTML element for the right job.

**Success Criterion - Optional Features**
Remove optional HTML tags (which aren't required for rendering), attribute quotes, or attributes that are set to their default value.

**Success Criterion - Avoid Non-standard Code**
Avoid using non-standard elements or attributes.

**Success Criterion - Custom Code**
Only use custom elements or Web Components if you cannot utilize native HTML elements or if you need tightly regulated control over the implementation of design system components.

## Proposed text

HTM semantics are important. They don't just play a key role in making the Web look the way it does, they have a function in accessibility, SEO, and even in sustainability. Ensuring that you markup your content correctly and avoid cluttering your markup wastefully will reduce emissions.

**Success Criterion - Semantic Code**
Content must be marked up semantically using the right HTML element for the right job.

**Success Criterion - Optional Features**
Remove optional HTML tags, attribute quotes, and default attributes only when they do not negatively impact functionality, accessibility, or readability. Retain them when they enhance accessibility, maintain clarity, or ensure consistent browser rendering.

**Success Criterion - Avoid Non-standard Code**
Avoid using non-standard elements or attributes.

**Success Criterion - Prefer standard HTML over custom elements or web components**
Prefer using standard HTML elements and attributes. Only use custom elements or Web Components if you cannot utilize native HTML elements or if you need tightly regulated control over the implementation of design system components. Ensure that the use of custom elements or Web Components do not negatively impact accessibility. Use appropriate ARIA attributes when necessary to convey semantic information to assistive technologies.

## Reasons

1."Success Criterion - Optional Features" The original text is too strict and potentially harmful. 
2. "Success Criterion - Custom Code". Custom elements or Web Components should never compromise accessibility. Developers must ensure that the custom elements behave correctly for users who rely on assistive technology.

**Comment by hidde** [comment]
My thoughts: 

1. You changed “HTML semantics” to “HTM semantics”, I guess typo
2. I wonder if we also wanted to  update the title “Use HTML Elements Correctly”, something like “Optimize use of HTML elements”?
3. I like your addition of “Retain them when they enhance accessibility, maintain clarity, or ensure consistent browser rendering.” but wonder if it applies to almost all cases? Eg frameworks that produce a lot of classes, like Tailwind, could be said to “maintain clarity” for someone (the developer) or even consistent browser rendering? 
4. Would not add “Ensure that the use of custom elements or Web Components do not negatively impact accessibility. Use appropriate ARIA attributes when necessary to convey semantic information to assistive technologies.”; I agree with the advice, but ARIA and WCAG already cover this.
5. While we're in this, section, may I suggest another reword? To me, this feels a bit vague: “Content must be marked up semantically using the right HTML element for the right job.” What is right? When it comes to HTML, whatever the WHATWG's HTML living standard says is the “[intended semantic purpose](https://html.spec.whatwg.org/dev/dom.html#semantics-2:~:text=intended%20semantic%20purpose)”, but I feel it would help to go broader. Eg, for some meanings of ”semantics”, semantics for web content are described the ARIA spec. What about “Content must be marked up according to relevant standards.” ?
